router-tests/testenv/testenv.go

```diff
@@ -359,17 +379,17 @@ func createTestEnv(t testing.TB, cfg *Config) (*Environment, error) {
 		metricReader = metric.NewManualReader()
 		cfg.MetricReader = metricReader
 		cfg.MetricOptions.EnableOTLPRouterCache = true
-		if cfg.ModifyRouterConfig == nil {
-			cfg.ModifyRouterConfig = func(cfg *nodev1.RouterConfig) {
-				cfg.FeatureFlagConfigs = nil
-			}
-		} else {
-			old := cfg.ModifyRouterConfig
-			cfg.ModifyRouterConfig = func(cfg *nodev1.RouterConfig) {
-				old(cfg)
-				cfg.FeatureFlagConfigs = nil
-			}
-		}
+		//if cfg.ModifyRouterConfig == nil {
```

Comment:
Mistake?

Reply:
Forgot to delete that when I updated the assertion logic for the cache hits.